handbook/people-ops/onboarding/before_starting.md

```diff
@@ -0,0 +1,43 @@
+# Before starting (onboarding preparation)
+
+This is a breakdown of what new teammates (and hiring managers) can expect People Ops to do to help them prepare for their first day at Sourcegraph. 
+
+## Future improvements
+
+It’s important to note that today’s onboarding procedures were created and built for a much smaller volume of people than we are currently onboarding. This page will outline the steps we are currently taking for each weekly cohort, but please keep in mind that many of these procedures and tools will be changing as we our first Onboarding and Training Manager starts. 
+
+Recent and planned improvements are noted in _italics_ in the list below.
+
+## People Ops onboarding
+
+### Immediately after offer signing
+
+- Each new teammate is added to [teammate tracking spreadsheet](https://docs.google.com/spreadsheets/d/1qt3rGNRVaSyi_N2uN3DaXvlg8LjfKLJb8PFfrQsmR-I/edit#gid=948830545) (name, title, start date, location)
+
+### One week prior to first day
+
+- Each new teammate receives a welcome email a week before their start date providing information and asking questions about benefits, email address selection, equipment purchases, and other onboarding logistics (we provide two templates, for US and non-US teammates)
+  - [US template](https://docs.google.com/document/d/1a3uhkt0bvmMuTmFBvroEXSdQGIPwCz8BLgk_YiSMZUQ/edit)
+  - [Non-US template](https://docs.google.com/document/d/1OxrGknE6NKoeMY7qMDe5th8ZL3juxFOKfH-eywWuwko/edit)
+  - _As of 2021-01-18, all hiring managers are CCed on these emails_
```

Comment:
🎉

Reply:
I was making a couple of edits to our buddy page but those are done now and I'll add in that section here too!